products.json

```
@@ -1,33 +1,54 @@
 {
     "androidstudio": {
-        "folder-name": "AndroidStudio",
-        "preferences-path": "~/Library/Application Support/Google/"
+        "display_name": "Android Studio",
+        "folder_name": "AndroidStudio",
+        "preferences_path": "~/Library/Application Support/Google/",
+        "bundle_id": "com.google.android.studio",
+        "uid": "92b0f147-c697-4f0b-b10a-10d823534408"
     },
     "appcode": {
-        "folder-name": "AppCode"
+        "folder_name": "AppCode",
+        "bundle_id": "com.jetbrains.appcode",
+        "uid": "d813b1dd-ee4a-470f-8c4d-73a55248e1b5"
     },
     "clion": {
-        "folder-name": "CLion"
+        "folder_name": "CLion",
+        "bundle_id": "com.jetbrains.clion",
+        "uid": "51a5ab39-dad4-47da-90af-a59be0f3b960"
     },
     "datagrip": {
-        "folder-name": "DataGrip"
+        "folder_name": "DataGrip",
+        "bundle_id": "com.jetbrains.datagrip",
+        "uid": "cd332057-2b16-4243-bbdf-a7d63718cf44"
     },
     "goland": {
-        "folder-name": "GoLand"
+        "folder_name": "GoLand",
+        "bundle_id": "com.jetbrains.goland",
+        "uid": "4d578e7e-c467-4e5c-9126-977f2226cf9e"
     },
     "idea": {
-        "folder-name": "IntelliJIdea"
+        "folder_name": "IntelliJIdea",
```

The official product name is "IntelliJ IDEA"

```suggestion
        "display_name": "IntelliJ IDEA",
        "folder_name": "IntelliJIdea",
```